samples/RNWinRTTestApp/windows/ExperimentalFeatures.props

```diff
@@ -46,8 +46,11 @@
       -include Windows.Security.Cryptography.BinaryStringEncoding
       -include Windows.UI.Notifications
       -include Windows.Data.Xml.Dom
+      -include Windows.Networking.ServiceDiscovery.Dnssd
+      -include Windows.Devices.Enumeration
+      -include System.Devices.AepService
+          <!-- TO DO: Remove AepService if not needed -->
```

Comment:
Is this AepService potentially related to milestone two?

Reply:
Used AepService to define 'protocolId' for the AQS query, even in `findAllDevicesAsync`.

However, we're not really using the projected type here directly, just telling WinRT to assign the protocol ID to a property of type in AepService namespace. I'm yet to check if removing this doesn't break scanning. Will do that and remove if not needed.